setup.cfg.in

```diff
@@ -3,6 +3,7 @@ version = @VINEYARD_VERSION@
 
 [options]
 install_requires =
+    jinja2
     numpy
```

Comment:
The `kubernetes` package has been used as well, add it to the dependencies please.